composer.json

```diff
@@ -1,41 +1,46 @@
 {
-    "name":         "knplabs/knp-gaufrette-bundle",
-    "type":         "symfony-bundle",
-    "description":  "Allows to easily use the Gaufrette library in a Symfony project",
-    "keywords":     ["file", "filesystem", "media", "abstraction"],
-    "homepage":     "http://knplabs.com",
-    "license":      "MIT",
-    "authors":      [
-        {
-            "name":     "Antoine Hérault",
-            "email":    "antoine.herault@gmail.com"
-        },
-        {
-            "name":     "The contributors",
-            "homepage": "https://github.com/knplabs/KnpGaufretteBundle/contributors"
-        }
-    ],
-    "require":      {
-        "php": ">=7.1",
-        "knplabs/gaufrette": "^0.11",
-        "symfony/config": "^4.4|^5.0|^6.0",
-        "symfony/dependency-injection": "^4.4|^5.0|^6.0",
-        "symfony/http-kernel": "^4.4|^5.0|^6.0"
+  "name": "knplabs/knp-gaufrette-bundle",
+  "type": "symfony-bundle",
+  "description": "Allows to easily use the Gaufrette library in a Symfony project",
+  "keywords": [
+    "file",
+    "filesystem",
+    "media",
+    "abstraction"
+  ],
+  "homepage": "http://knplabs.com",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Antoine Hérault",
+      "email": "antoine.herault@gmail.com"
     },
-    "require-dev": {
-        "symfony/phpunit-bridge": "^5.4",
-        "symfony/console": "^4.4|^5.0|^6.0",
-        "symfony/filesystem": "^4.0|^5.0|^6.0",
-        "symfony/yaml": "^4.0|^5.0|^6.0"
-    },
-    "autoload":     {
-        "psr-4": {
-            "Knp\\Bundle\\GaufretteBundle\\": ""
-        }
-    },
-    "extra": {
-        "branch-alias": {
-            "dev-master": "0.8.x-dev"
-        }
+    {
+      "name": "The contributors",
+      "homepage": "https://github.com/knplabs/KnpGaufretteBundle/contributors"
+    }
+  ],
+  "require": {
+    "php": ">=7.1",
+    "knplabs/gaufrette": "^0.11",
+    "symfony/config": "^4.4|^5.0|^6.0|^7.0",
```

Comment:
Symfony 4 should be removed: supporting three major versions of Symfony is enough

Reply:
Ok, SF5 requires PHP 7.2.5 so i updated php requirement too.